_studio/mfx_lib/shared/include/mfx_h264_encode_struct_vaapi.h

```diff
@@ -314,6 +314,14 @@ typedef struct tagENCODE_CAPS
     UCHAR    MaxNum_WeightedPredL1;
 } ENCODE_CAPS;
 
+// We cannot modify ENCODE_CAPS structure, but need to add new features
+struct MFX_ENCODE_CAPS
+{
+    ENCODE_CAPS ddi_caps;
+    bool CQPBRCSupport;
```

Comment:
A little disproportion in names... You have ```BRC``` in first name but not in two subsequent. I suggest either remove it from first one or add to the rest to align naming style

Reply:
CBR and VBR contains BR in name of method, but CQP not contains this one, in this way i added brc suffix. If you think what this will more clearly, i will change it.

Reply:
Aligned with first.

Reply:
"constant bit rate bit rate control" sounds ugly. It was clear without BRC what is it about. Common comment would be enough.

Reply:
@lakulako I am confused. You think I must revert changes (CQPBRC, CBR, VBR) and add comments?

Reply:
Don't use BRC at all and add common comment like "Bit rate control capabilities, provided by MSDK"

Reply:
Done.